src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,11 +5,26 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private double loadCapacity;
```

Comment:
The field 'loadCapacity' should have an access modifier to encapsulate the field properly. Consider using 'private' to restrict direct access to the field and provide getters and setters for access.